aiida-registry-app/src/Components/Details.jsx

```diff
@@ -105,6 +106,15 @@ function Details({pluginKey}) {
             </p>
             )}
 
+            {value.warnings && (
```

Comment:
Sorry, while writing the email I realized one more small request.

If there are no warnings, can we just show a "sucess"-style message "All checks passed"?

Many plugins pass all checks, so developers would be looking for this section and not finding it

Reply:
Are there cases where warnings will be empty for another reason?

If so, we may need to track in the json somewhere whether the `fetch` stage was completed for the plugin

Reply:
> If there are no warnings, can we just show a "sucess"-style message "All checks passed"?

Done.

> Are there cases where warnings will be empty for a reason other than everything being fine (some error encountered, so the code does not get to the stage of adding warnings)?

Yes this is the case for `aiida-fenics` for example, the plugin link is broken, but even if the warnings is empty it will not display "All checks passed" because this depends on another condition that the detailed infomration must be fetched first. If not, nothing will be displayed.

Reply:
I think this is another issue, putting the registry checks on this section will not work for plugins that have empty `metadata` key. I'll try to fix this